src/test/infra/lttng/test_latency_definitions.py

```diff
@@ -1256,6 +1298,53 @@ def test_inter_proc(
 
         assert df.equals(df_expect)
 
+    def test_intra_proc_empty_data(
+        self,
+        mocker,
+        create_lttng,
+        create_publisher_lttng,
+        setup_bridge_get_publisher,
+        create_publisher_struct,
+        bridge_setup_get_callback,
+        create_subscription_lttng,
+        create_subscription_struct,
+        create_comm_struct
+    ):
+        pub_handle = 7
+        callback_obj = 12
+
+        data = Ros2DataModel()
+        data.finalize()
+
+        publisher = create_publisher_struct()
+        subscription = create_subscription_struct()
+        callback = subscription.callback
+        communication = create_comm_struct(publisher, subscription)
+        publisher_lttng = create_publisher_lttng(pub_handle)
+        sub_cb_lttng = create_subscription_lttng(callback_obj, 5, callback_obj)
```

Comment:
magic number.

Reply:
addressed in :[c670001](https://github.com/tier4/CARET_analyze/pull/210/commits/c6700010b36b4d8e0ec50486f0f6b972bd11a59a)